### PR TITLE
Review

### PR DESCRIPTION
**Pull request by aze12345**
Anna, Jackson added swerve functionality to your code. Please look through this PR and make comments as needed (use up to date 2024 standard).

### CODE REVIEW

File: src/main/java/org/usfirst/frc4904/robot/RobotMap.java [hunk 1]
```diff
@@ -0,0 +1,268 @@
+package org.usfirst.frc4904.robot;
+
+import org.usfirst.frc4904.standard.custom.controllers.CustomCommandJoystick;
+import org.usfirst.frc4904.standard.custom.controllers.CustomCommandXbox;
+import org.usfirst.frc4904.standard.custom.motorcontrollers.CANTalonFX;
```
Comment (Papayapaya) [inline src/main/java/org/usfirst/frc4904/robot/RobotMap.java:5]: this is bad

File: src/main/java/org/usfirst/frc4904/robot/RobotMap.java [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+import edu.wpi.first.wpilibj.SerialPort;
+import edu.wpi.first.wpilibj.livewindow.LiveWindow;
+
+public class RobotMap {
+
+    public static class Port {
+        // public static class Network {
+        //     public static SocketAddress LOCAL_SOCKET_ADDRESS = new InetSocketAddress(3375);
+        //     public static SocketAddress LOCALIZATION_ADDRESS = new InetSocketAddress("10.49.04.10", 4321);
+        // }
```
Comment (annvoor) [inline src/main/java/org/usfirst/frc4904/robot/RobotMap.java:46]: why are these here

File: src/main/java/org/usfirst/frc4904/robot/RobotMap.java [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+
+       }
+
+    public static class Metrics {
+        // blinky constants
+        // public static class Chassis {
+        //     public static final double GEAR_RATIO = 69/5;
+        //     public static final double WHEEL_DIAMETER_METERS = Units.inchesToMeters(4);
+        //     public static final double TRACK_WIDTH_METERS = 0.59;
+        // }
```
Comment (annvoor) [inline src/main/java/org/usfirst/frc4904/robot/RobotMap.java:90]: don't think we need these anymore

File: src/main/java/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java [hunk 4]
```diff
@@ -0,0 +1,86 @@
+package org.usfirst.frc4904.robot.humaninterface.drivers;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.standard.humaninput.Driver;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+
+public class NathanGain extends Driver {
+	public static boolean isFlippy = false;
```
Comment (Papayapaya) [inline src/main/java/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java:9]: here and 26-27 static varible names should be upper snake case

File: src/main/java/org/usfirst/frc4904/robot/RobotMap.java [hunk 5]
```diff
@@ -0,0 +1,268 @@
...
+import org.usfirst.frc4904.standard.custom.controllers.CustomCommandXbox;
+import org.usfirst.frc4904.standard.custom.motorcontrollers.CANTalonFX;
+
+//import com.ctre.phoenix.motorcontrol.NeutralMode; //broken
+import com.kauailabs.navx.frc.AHRS;
+
+//imports for rev robotics neo 550s
+import com.revrobotics.CANSparkMax;
+
+//import com.ctre.phoenix.motorcontrol.InvertType; //broken
```
Comment (annvoor) [inline src/main/java/org/usfirst/frc4904/robot/RobotMap.java:13]: import com.ctre.phoenix.motorcontrol.InvertedValue;

File: src/main/java/org/usfirst/frc4904/robot/RobotMap.java [hunk 6]
```diff
@@ -0,0 +1,268 @@
+package org.usfirst.frc4904.robot;
+
+import org.usfirst.frc4904.standard.custom.controllers.CustomCommandJoystick;
+import org.usfirst.frc4904.standard.custom.controllers.CustomCommandXbox;
+import org.usfirst.frc4904.standard.custom.motorcontrollers.CANTalonFX;
+
+//import com.ctre.phoenix.motorcontrol.NeutralMode; //broken
```
Comment (annvoor) [inline src/main/java/org/usfirst/frc4904/robot/RobotMap.java:7]: import com.ctre.phoenix6.signals.NeutralModeValue;

File: src/main/java/org/usfirst/frc4904/robot/Robot.java [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+    }
+
+    @Override
+    public void disabledInitialize() {
+        //do things like setting brake mode here
+    }
+
+    @Override
+    public void disabledExecute() {
+    }
```
Comment (Papayapaya) [inline src/main/java/org/usfirst/frc4904/robot/Robot.java:103]: this seems important to have for safety

File: src/main/java/org/usfirst/frc4904/robot/RobotMap.java [hunk 8]
```diff
@@ -0,0 +1,268 @@
...
+        Component.FLturn = new CustomCANSparkMax(Port.CANMotor.FRONT_LEFT_TURN, MotorType.kBrushless, false);
+        Component.FRdrive  = new CANTalonFX(Port.CANMotor.FRONT_RIGHT_DRIVE);
+        Component.FRturn = new CustomCANSparkMax(Port.CANMotor.FRONT_RIGHT_DRIVE, MotorType.kBrushless, false);
+        Component.BLdrive  = new CANTalonFX(Port.CANMotor.BACK_LEFT_DRIVE);
+        Component.BLturn = new CustomCANSparkMax(Port.CANMotor.BACK_LEFT_TURN, MotorType.kBrushless, false);
+        Component.BRdrive  = new CANTalonFX(Port.CANMotor.BACK_RIGHT_DRIVE);
+        Component.BRturn = new CustomCANSparkMax(Port.CANMotor.BACK_RIGHT_TURN, MotorType.kBrushless, false);
+
+
+        // Component.backRightWheelTalon.setSafetyEnabled(false);
```
Comment (annvoor) [inline src/main/java/org/usfirst/frc4904/robot/RobotMap.java:236]: are these from west coast drive? can we just delete them
